src/final2/subtests/InvalidCommandTest.java

```diff
@@ -155,17 +155,11 @@ public void createExistingAntTest() {
 	@Test
 	public void invalidAntNameTest() {
 		for (String invalidAntName : INVALID_ANT_NAMES) {
-			inputFile = new String[] {
-					"0000",
-					"0000",
-					"0a00",
-					"0000"
-			};
 			runs = new Run[] {
 					new ErrorRun("create " + invalidAntName + ",1,1"),
 					quit()
 			};
-			sessionTest(runs, Input.getFile(inputFile));
+			sessionTest(runs, Input.getFile(TASK_SHEET_INPUT_FILE_1));

```

Comment:
I'd rather revert this. But in the escape test, any well-formed playing field is okay, so I'd keep TASK_SHEET_INPUT_FILE_1 there.